File: equipment-management-frontend/src/app/DTO/TicketDTO.ts
import {EtatTicket} from "../models/Ticket";

export  interface TicketDTO{
  id: number;
  description: string;
  etatTicket: string;
  userId: number;
  technicienId: number | null; 
  panneId: number;
  newTechnicianId?: number;
  newStatus?: EtatTicket;
  dateCreation: string;
}

File: equipment-management/src/main/java/com/itsolutions/equipment_management/DTO/TicketDTO.java
package com.itsolutions.equipment_management.DTO;

import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@NoArgsConstructor
@AllArgsConstructor
@Setter
// TicketDTO.java
public class TicketDTO {
    private Long id;
    private String description;
    private String etatTicket;
    private Long panneId;
    private Long userId;
    private Long technicienId;

}
